perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_virt_eduPersonScopedAffiliationsTest.java

```diff
@@ -265,4 +271,44 @@ public void resolveAttributeValueChangeTest() throws Exception {
 		assertEquals(auditEvents.get(0).getClass(), AttributeChangedForUser.class);
 	}
 
+	@Test
+	public void lastAccessValid() throws Exception {
+		ues1.getExtSource().setType(ExtSourcesManager.EXTSOURCE_IDP);
+		ues1.setLastAccess(LocalDateTime.now().minusYears(10).format(Utils.lastAccessFormatter));
```

Comment:
Nitpick - shouldn't we set this value based on the current config value of idpLoginValidity, just in case it changes in the future? Or change idpLoginValidity for the test and then return it to the original value; it doesn't matter.